TreeCoverLossAnalysis.pyt

```diff
@@ -372,9 +393,27 @@ class TreeCoverLossAnalysis(object):
                     "Name": "geotrellis-treecoverloss-cores",
                     "Market": "SPOT",
                     "InstanceRole": "CORE",
```

Comment:
this should now be on-demand to

Reply:
I pushed an update that changed the core instances to on-demand.